src/basic/virt.c

```diff
@@ -21,6 +21,14 @@
 #include "string-util.h"
 #include "virt.h"
 
+#if defined(__i386__) || defined(__x86_64__) || defined(__arm__) || defined(__aarch64__)
```

Comment:
this conditionalization must be dropped, you use the enum further down in all cases

Reply:
Whoops. Sorry about that. Fixed.